earth_enterprise/rpms/build.gradle

```diff
@@ -247,14 +246,10 @@ task openGeePostGisRpm(type: GeeRpm) {
     }
 }
 
-if (project.hasProperty('buildOpenGee')) {
-    openGeePostGisRpm.dependsOn stageOpenGeeInstall
-}
```

Comment:
`setGeeVersion` task now takes care of this dependency.

Reply:
I think that's confusing.  I wouldn't expect that setting a version would do all the build work necessary for packaging.

Reply:
If you find it confusing I could put these back but they will have no effect based on current code.